api/mesh/v1alpha1/dataplane_helpers.go

```diff
@@ -13,13 +13,21 @@ const (
 	// Mandatory tag that has a reserved meaning in Kuma.
 	ServiceTag     = "kuma.io/service"
 	ServiceUnknown = "unknown"
-	// Mandatory tag that has a reserved meaning in Kuma.
-	ZoneTag = "kuma.io/zone"
+
+	// Locality related tags
+	RegionTag   = "kuma.io/region"
+	ZoneTag     = "kuma.io/zone"
+	SubZoneTag  = "kuma.io/sub-zone"
+	LocalityTag = "kuma.io/locality" // optionally set to `disable`
```

Comment:
I've discussed this with Marco yesterday. I presume this tag is to disable this feature?

I think this feature should be enabled for now with Mesh
```
type: Mesh
routing:
  localityAwareLoadBalancing: true
```

and eventually should be moved (not in this PR) to explicit TrafficRoute policy that will be also used to pick load balancing strategy